src/main/java/Duke.java

```diff
@@ -1,10 +1,186 @@
+import java.util.Scanner;
+
 public class Duke {
```

Comment:
Perhaps the class name can be changed to match the name of your chat bot? To maintain consistency and make it more intuitive when the chatbot is called.